hs_swat_modelinstance/page_processors.py

```diff
@@ -0,0 +1,125 @@
+__author__ = 'Mohamed Morsy'
+from mezzanine.pages.page_processors import processor_for
+from crispy_forms.layout import Layout, Fieldset, ButtonHolder, Submit, HTML
+from hs_core import page_processors
+from hs_core.views import *
+from forms import *
```

Comment:
also is `from forms` a relative import? use the explicit relative import syntax `from .forms import foo, bar, baz`